lib/beaker-answers/versions/version20162.rb

```diff
@@ -26,7 +28,55 @@ def generate_answers
 
       the_answers[console.name].merge!(orchestrator_db)
 
-      the_answers
+      if @type == :bash
```

Comment:
This handling feels a little strange.  How about case on type and hand off two two different functions generate_bash_answers and generate_hiera_config